projects/observability/src/shared/graphql/request/handlers/entities/query/topology/entity-topology-graphql-query-handler.service.ts

```diff
@@ -123,8 +123,10 @@ export class EntityTopologyGraphQlQueryHandlerService
     }
   }
 
-  public convertResponse(response: TopologyServerResponse, request: GraphQlEntityTopologyRequest): EntityNode[] {
-    return this.convertToEntityNodes(request, response.results);
+  public convertResponse(response: unknown, request: GraphQlEntityTopologyRequest): EntityNode[] {
```

Comment:
why did this lose typing?

Reply:
We have used this pattern at a few places. We tend to avoid exporting "server" types from the handler but in this case, we must export them if we want the tests to be able to call this method. 

Here is an example:

```
  public convertResponse(response: unknown, request: GraphQlEntitiesQueryRequest): Observable<EntitiesResponse> {
    return this.entitiesGraphQlQueryBuilderService.buildResponse(response, request);
  }
```

Reply:
> we must export them if we want the tests to be able to call this method.

I thought it should work as long as we don't need a direct type reference?

Reply:
It complains. Probably because we didn't pass all properties. I didn't want to debug and change this in the code as part of this PR. Do you mind if we address all similar occurrences in a new PR?